articles/tensorflow-vs-pytorch/index.md

```diff
@@ -0,0 +1,113 @@
+Today, we have a lot of people using both TensorFlow and PyTorch. These are the two most popular DL frameworks. 
+
+Discussing these two popular frameworks will be the subject of our article today. But first, let's get to know what TensorFlow and PyTorch are.
+
+### What is TensorFlow (TF)?
+
+TensorFlow is a software application popular for implementing deep learning algorithms. The Google Brain team developed it, and it was released as an open-source platform in 2015. It supports both the R and Python programming languages. It was initially started as a research-based architecture for Google's in-house applications but was eventually open-sourced to be used commercially.
+
+It is known as "TensorFlow" because it takes in inputs as tensors or multi-dimensional arrays (MDAs). These tensors (MDAs) then flow through a system of operations and comes out the other end as outputs.
+
+The key features of TensorFlow include:
+
+1. It is actively maintained by the developers at Google. Currently, TF2.3 is the latest released version.
+2. It is open-source software that grants developers access to use and distribute the software for different purposes.
+3. It makes use of TensorBoard for visualizing model graphs, tracking and visualizing metrics, and displaying data such as images and texts.
+4. It is a flexible framework. It supports the experimentation of new deep learning models.
+
+An extra feature to note is that TensorFlow 2.x now adopts the Keras API as its default standard. This eliminates the need to import the Keras API onto TensorFlow separately.
+
+### What is PyTorch?
+
+PyTorch is a relatively new deep learning framework as it came out a year later after TensorFlow was released. The name PyTorch was inspired by a popular "[torch](http://torch.ch/)" deep learning framework, which was written in the [Lua](https://www.lua.org/) programming language. However, learning the Lua programming language was a big barrier, especially when learning deep learning. It also didn't offer the modularity necessary to interface with other deep learning libraries currently present. 
+
+This problem was solved by a team of AI researchers inspired by the torch's programming style. They decided to develop an improvised form of torch with the Python (py) programming language, which later became known as PyTorch. 
+
+Some notable features of PyTorch include:
+
+1. It is extremely fast and feels Native.
+2. It has native support for Python and all of its libraries. This feature makes it very easy to learn. 
+3. It's easy to use API that ensures easy usability when making use of the API.
+4. It is actively maintained and used by the developers at Facebook.
+5. It supports Nvidia's [CUDA](https://en.wikipedia.org/wiki/CUDA), ensuring that the code runs on a GPU, decreasing the time needed to run the code.
+6. Dynamic Computation Graphs is a key feature in PyTorch. It ensures that the graphs build up dynamically as you code.
```

Comment:
Explain what a dynamic computation graph is. Add a section on static graphs vs. dynamic graphs and how it affects the performance. This section forms the heart of this article.